README.md

```diff
@@ -47,16 +47,14 @@ Configuration
 Txbr requires several configuration options in order to communicate with Braze and Transifex. Configuration is done in the YAML file format. Here's an example:
 
 ```yaml
+transifex_api_username: tx_username
```

Comment:
Just curious why these moved up

Reply:
They were in the wrong place from the get-go :(